modules/apps/accessibility/accessibility-test/src/testFunctional/tests/AccessibilityMenu.testcase

```diff
@@ -35,29 +37,79 @@ definition {
 	}
 
 	@description = "LPS-178192. Verifies only if the setting has never been changed by the user, either when signed in or not signed in, will the default value be displayed.."
-	@ignore = "Test Stub"
 	@priority = 5
 	test BrowserLocalStorageSmoke {
 		property portal.acceptance = "true";
 
-		//task ("Given When Then") {
+		task ("Given created new user A") {
+			JSONUser.addUser(
+				userEmailAddress = "userea@liferay.com",
+				userFirstName = "userfnA",
+				userLastName = "userlnA",
+				userScreenName = "usersnA");
+		}
 
-		// // Given a new created user A
-		// // And signed out as guest user
-		// // When toggle  show underline to OFF state
-		// // And sign in as a new user for first time
-		// // And toggle show under to ON
-		// // And user signs out
-		// // Then
-		// // show underline is set to OFF
-		// // And When
-		// // guest user toggles show underline to ON
-		// // And guest user toggles show underline to OFF
-		// // And sign in as user A
-		// // Then show underline is set to ON state
+		task ("And signed out as guest user") {
+			SignOut.signOut();
+		}
 
-		//}
+		task ("When toggle show underline to OFF state") {
```

Comment:
Let's switch the OFF states to ON and vice versa. When creating the test scenarios at first, the default state was ON but then it was changed to OFF state. This change of state affects this test scenario.